src/label/Label.ts

```diff
@@ -45,12 +44,6 @@ export const LabelBase = ThemeableMixin(WidgetBase);
 
 @theme(baseCss)
 export default class Label extends LabelBase<LabelProperties>  {
-	@onPropertiesChanged()
-	protected onPropertiesChanged(event: PropertiesChangeEvent<this, LabelProperties>) {
```

Comment:
Ok cool. It worries me slightly that the Label will throw unless I pass a registry after landing this PR, so hopefully we can make the change to the registry mixin soon after.